apps/photos/src/components/PhotoFrame.tsx

```diff
@@ -82,6 +103,148 @@ const PhotoFrame = ({
     const [isSourceLoaded, setIsSourceLoaded] = useState(false);
     const [conversionFailed, setConversionFailed] = useState(false);
 
+    const avatarColors: string[] = [
```

Comment:
These colors need to be defined along side with other colors inside theme/ folder.